com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/RaytracingIntersection.hlsl

```diff
@@ -11,11 +11,37 @@ struct RayCone
     float spreadAngle;
 };
 
+// Structure that defines the current state of the visibility
+struct RayIntersectionDebug
+{
+    // Distance of the intersection
+    float t;
+    // Barycentrics of the intersection
+    float2 barycentrics;
```

Comment:
This doesn't seems to be used, is it useful? barycentrics

Reply:
Yes, it's for the future, a debug mode that we may want to use